app/models/reminder_activity.rb

```diff
@@ -1,2 +1,6 @@
 class ReminderActivity < Activity
+  def self.already_reminded_appointment_ids
```

Comment:
Seems legit. Though perhaps the `already_` can be dropped in favour of just `reminded_appointment_ids` as the 'already' part is implied by 'reminded'.